### PR TITLE
MF1933099 Homework 5 更新后

### PR DESCRIPTION
**Pull request by RLLDLBF**
From MF1933099 卫昱阳

- LinkedList,2   same
- Arrays,242     same
- ArrayList,2     same
- ArrayList,118 same
- Integer,102    same

> 注：其中Arrays,242的Java测试用例为 public static void sort (int[] a) 的

### CODE REVIEW

**Comment by wangyue97** [comment]
MF1933098王悦评估已通过

File: MF1933099/swift/ArraySlice_popFirst.swift [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+*  @apiSignature: ArraySlice$mutating func popFirst() -> Element?
+*  @description: Test swift api ArraySlice$mutating func popFirst() -> Element?
+*  @Map: java.util.LinkedList$ public E pollFirst ()
+*/
+import Foundation
+
+class ArraySlice_popFirst{
+    /**
+    * input: 1
+    * class0 ArraySlice$class0: ArraySlice = ["Nanjing", "University", "", "Software", "Engineering", "Group"]
```
Comment (idiomaticrefactoring) [inline MF1933099/swift/ArraySlice_popFirst.swift]: class0 去掉：后面的

File: MF1933099/swift/Array_makeIterator.swift [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+*  @description: Test swift api Array$func makeIterator() -> IndexingIterator<Array<Element>>
+*  @Map: java.util.ArrayList$ public Iterator iterator ()
+*/
+import Foundation
+
+class Array_makeIterator{
+    /**
+    * input: 1
+    * class0 Array$cclass0 = ["Nanjing", "University", "", "Software", "Engineering", "Group"]
+    * output: 1
```
Comment (idiomaticrefactoring) [inline MF1933099/swift/Array_makeIterator.swift]: cclass0?

**Review by idiomaticrefactoring — CHANGES_REQUESTED** [review]
perfect-2

**Pull request closed without merging**